subways/structure/route.py

```diff
@@ -33,24 +33,27 @@
 DISALLOWED_ANGLE_BETWEEN_STOPS = 20  # in degrees
 
 
-def get_start_end_times(
+def parse_time_range(
     opening_hours: str,
-) -> tuple[tuple[int, int], tuple[int, int]] | tuple[None, None]:
+) -> tuple[tuple[int, int], tuple[int, int]] | None:
     """Very simplified method to parse OSM opening_hours tag.
     We simply take the first HH:MM-HH:MM substring which is the most probable
     opening hours interval for the most of the weekdays.
     """
     start_time, end_time = None, None
     m = START_END_TIMES_RE.match(opening_hours)
-    if m:
-        ints = tuple(map(int, m.groups()))
-        start_time = (ints[0], ints[1])
-        end_time = (ints[2], ints[3])
+    if not m:
```

Comment:
just in case, do you handle "extended hour syntax", e.g. `Mo 20:00-26:00`?
https://wiki.openstreetmap.org/wiki/Key:opening_hours#:~:text=opening_hours%3D-,Mo%2020%3A00%2D26%3A00,-Open%20on%20Monday

Reply:
There's also "24/7", do we handle such case?

https://wiki.openstreetmap.org/wiki/Key:opening_hours#:~:text=Applied%20to%3A%20any%20non%2Dstop%20facilities%20(related%20tags)%2C%2024%20hours%20a%20day%2C%207%20days%20a%20week

Reply:
Yes. Such intervals are considered valid, see test case:
https://github.com/alexey-zakharenkov/subways/blob/process-route-duration/subways/tests/test_route.py#L48

Moreover, interval 15:00-02:00 is converted to 15:00-26:00 in GTFS:
https://github.com/alexey-zakharenkov/subways/blob/process-route-duration/subways/processors/gtfs.py#L282

Reply:
> There's also "24/7", do we handle such case?
> 
Good point! Added functionality and tests.